### PR TITLE
Confs to envs

### PR DESCRIPTION
**Pull request by zalbiraw**
- Update Tyk products versions
- Remove depends_on from docker-compose
- Replace config files with environment variables

### CODE REVIEW

**Comment by marksou** [comment]
@zalbiraw Let me know when I can test this as I have an Apple silicon Mac, so can test the Arm containers 😀

File: confs/pump.env [hunk 1]
```diff
@@ -0,0 +1,30 @@
+TYK_PMP_OMITCONFIGFILE=true
```
Comment (sedkis) [inline confs/pump.env:1]: Is this a real env variable? I can't find it anywhere

https://github.com/TykTechnologies/tyk-pump/blob/master/config.go

Comment (sedkis) [reply confs/pump.env:1]: [Found it](https://github.com/TykTechnologies/tyk-pump/commit/73d4d3f88740), it's not released yet.  Let's keep it in.

File: confs/tyk_analytics.env [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+TYK_DB_UI_LOGINPAGE=
+TYK_DB_UI_NAV=dont_show_admin_sockets:false,hide_activity_by_api_section:false,hide_geo:false,hide_licenses_section:false,hide_logs:false,hide_tib_section:false
+TYK_DB_UI_UPTIME=
+TYK_DB_UI_PORTALSECTION=
+TYK_DB_UI_DESIGNER=
+TYK_DB_UI_DONTSHOWADMINSOCKETMESSAGES=false
+TYK_DB_UI_DONTALLOWLICENSEMANAGEMENT=false
+TYK_DB_UI_DONTALLOWLICENSEMANAGEMENTVIEW=false
+TYK_DB_HOMEDIR=/opt/tyk-dashboard
+TYK_DB_USESHARDEDANLAYTICS=false
```
Comment (armujahid) [inline confs/tyk_analytics.env:70]: There's a type in `TYK_DB_USESHARDEDANALYTICS` (ANLAYTICS should be ANALYTICS)

**Pull request merged**